packages/catalog-server/src/lib/firestore/firestore-repository.ts

```diff
@@ -160,16 +169,22 @@ export class FirestoreRepository implements Repository {
           // collection first.
           const elements = await t.get(customElementsRef);
 
+          const isLatest = versionDistTags.includes('latest');
+
           // Update the PackageVersion doc
-          await t.update(versionRef, {
+          // We remove the converter to fix the types:
```

Comment:
Not sure I understand "remove the converter" here, since it's actually adding "withConverter" -- seems like the opposite? :)